extension-manifest-v3/src/pages/panel/views/main.js

```diff
@@ -11,50 +11,35 @@ import Company from './company.js';
 const NOTIFICATIONS = [
   {
     icon: 'help',
-    text: msg`Ghostery is not fully functional because of the browser’s restrictions and missing additional permissions.`,
+    text: 'Ghostery is not fully functional because of the browser’s restrictions and missing additional permissions.',
     actions: [
       {
-        text: msg`Get Help`,
-        url: 'https://www.ghostery.com/support/',
+        text: 'Get help',
+        url: 'https://www.ghostery.com/support?utm_source=gbe',
       },
     ],
   },
   {
     icon: 'heart',
-    text: msg`Hey, do you enjoy Ghostery and want to support our work?`,
+    text: 'Hey, do you enjoy Ghostery and want to support our work?',
     actions: [
       {
-        text: msg`Become a contributor`,
-        url: 'https://www.ghostery.com/become-a-contributor/',
+        text: 'Become a Contributor',
+        url: 'https://www.ghostery.com/become-a-contributor?utm_source=gbe',
       },
     ],
   },
 ];
 
-const UNPROTECTED_SITES_URL = chrome.runtime.getURL(
-  '/pages/options/index.html',
-);
 const OPTIONS_URL = chrome.runtime.getURL('/pages/options/index.html');
 const ONBOARDING_URL = chrome.runtime.getURL('/pages/onboarding/index.html');
 
-function getPauseNotification(domain, paused, pauseType) {
+function getPauseNotification(paused) {
```

Comment:
as this is so simplified, we can remove the function and put the condition in-place.